bootstrap-page.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<head>
+    <title>Bootstrap Page</title>
+    <style>
+        .container {
+            float: left;
+            width: 50%;
+        }
+        .container-2 {
+            float: right;
+            width: 50%;
+        }
+        .carbon-ad {
+            float: left;
+            width: 100%;
+            border: 3px;
+        }
+        .container-3 {
+            float: none;
+            width: 50%;
+        }
+        ul {
+            list-style-type: none;
+        }
+        li {
+            display: inline;
+        }
+        .button {
+            display: linline;
+        }
+        .footer {
+            width: 25%;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <nav>
+            <ul>
+                <li><a href="#logo">Logo</a></li>
+                <li><a href="#home">Home</a></li>
+                <li><a href="#Docs">Docs</a></li>
+                <li><a href="#Examples">Examples</a></li>
+                <li><a href="#Icons">Icons</a></li>
+                <li><a href="#Themes">Themes</a></li>
+                <li><a href="#Blog">Blog</a></li>
+                <li><a href="#Github">Github icon</a></li>
+                <li><a href="#Twitter">Twitter icon</a></li>
+                <li><a href="#Open-Collective">Open Collective icon</a></li>
+                <li><button type="button">Download</button></li>                
+            </ul>
+        </nav>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="column">
+                <h1>Build fast, responsive sites with Bootstrap</h1>
+                <p>Quickly design and customize responsive mobile-first sits with Bootstrap, the world’s most popular front-end open source toolkit, featuring Sass variables and mixins, responsive grid system, extensive prebuilt components, and powerful JavaScript plugins.</p>
+                <div class="button">
+                    <button type="button">Download Examples</button>
```

Comment:
Put line 60 and 63 into one `div`.
Then if you wish to place the two together, you simply would style that `div`.